x-pack/solutions/security/plugins/security_solution/server/lists_integration/endpoint/validators/base_validator.test.ts

```diff
@@ -291,5 +303,136 @@ describe('When using Artifacts Exceptions BaseValidator', () => {
+      });
+
+      it('should do nothing if feature flag is turned off', async () => {
+        authzMock.canManageGlobalArtifacts = false;
+        setSpaceAwarenessFeatureFlag('disabled');
+
+        await expect(
+          validator._validateCanCreateGlobalArtifacts(exceptionLikeItem)
+        ).resolves.toBeUndefined();
+      });
+
+      it('should error is user does not have new global artifact management privilege', async () => {
+        authzMock.canManageGlobalArtifacts = false;
+
+        await expect(
+          validator._validateCanCreateGlobalArtifacts(exceptionLikeItem)
+        ).rejects.toThrow(noAuthzToManageGlobalArtifactsError);
+      });
+
+      it('should allow creation of global artifacts when user has privilege', async () => {
+        await expect(
+          validator._validateCanCreateGlobalArtifacts(exceptionLikeItem)
+        ).resolves.toBeUndefined();
+      });
+    });
+
+    describe('#validateCanUpdateItemInActiveSpace()', () => {
+      let savedExceptionItem: ExceptionListItemSchema;
+
+      beforeEach(() => {
+        savedExceptionItem = createExceptionListItemMock({
+          // Saved item is owned by different space id
+          tags: [buildPerPolicyTag('123'), buildSpaceOwnerIdTag('foo')],
+        });
+      });
+
+      it('should do nothing if feature flag is turned off', async () => {
+        setSpaceAwarenessFeatureFlag('disabled');
+        authzMock.canManageGlobalArtifacts = false;
+
+        await expect(
+          validator._validateCanUpdateItemInActiveSpace(exceptionLikeItem, savedExceptionItem)
+        ).resolves.toBeUndefined();
+      });
+
+      it('should error if updating a global item when user does not have global artifact privilege', async () => {
+        authzMock.canManageGlobalArtifacts = false;
+        savedExceptionItem.tags = [GLOBAL_ARTIFACT_TAG, buildSpaceOwnerIdTag('foo')];
+
+        await expect(
+          validator._validateCanUpdateItemInActiveSpace(exceptionLikeItem, savedExceptionItem)
+        ).rejects.toThrow(noAuthzToManageGlobalArtifactsError);
+      });
+
+      it('should error if updating an item outside of its owner space id when user does not have global artifact privilege', async () => {
+        authzMock.canManageGlobalArtifacts = false;
+
+        await expect(
+          validator._validateCanUpdateItemInActiveSpace(exceptionLikeItem, savedExceptionItem)
+        ).rejects.toThrow(itemCanNotBeManagedInActiveSpaceErrorMessage);
+      });
+
+      it('should allow updates to global items when user has global artifact privilege', async () => {
+        savedExceptionItem.tags = [GLOBAL_ARTIFACT_TAG, buildSpaceOwnerIdTag('foo')];
+
+        await expect(
+          validator._validateCanUpdateItemInActiveSpace(exceptionLikeItem, savedExceptionItem)
+        ).resolves.toBeUndefined();
+      });
+
+      it('should allow update to item outside of owner space id when user has global artifact privilege', async () => {
```

Comment:
i think this test case is missing:
`should allow update to item inside of owner space id when user has no global artifact privilege`